evap/static/css/_quick-review.scss

```diff
@@ -67,6 +67,9 @@
     &:focus {
         outline: none;
     }
+    &[data-layer="0"] {
```

Comment:
what does this do? Github renders it red as if it was completely broken xD

Reply:
its a selector for all elements with the `data-layer` attribute set to `0`. it's correct, don't mind github.